docs/services/data/configure-data-capture.md

```diff
@@ -227,7 +234,7 @@ The following example captures data from two analog readers that provide a volta
               {
                 "method": "Analogs",
                 "capture_frequency_hz": 1,
-                "name": "rdk:component:board/board",
+                "name": "rdk:component:board/my-esp32",
                 "additional_params": {
                   "A2": "",
```

Comment:
This should be grabbing the analog reader by name, so this should be "sensor"

Under the hood, this is calling [AnalogReaderByName](https://github.com/viamrobotics/rdk/blob/d292ecc835cab4dc88c2e51e6529d40766111319/components/board/board.go#L56)

Reply:
Ooh i will change "sensor" to A1 above and add an A2

Reply:
Also should I change the pin numbers to be actual ADC capable pins--pin 34 wouldn't work would it? Or does it not matter because we're using an external ADC?

Reply:
Don't have board-specific configuration answers — just know that if they're configured in a way that works at RDK reconfiguration time, they can have data captured off of them.